README.md

```diff
@@ -15,86 +15,29 @@ reports on images stored in Harbor registry as part of its vulnerability scan fe
 
 ## TOC
 
-- [Getting started](#getting-started)
-  - [Prerequisites](#prerequisites)
-  - [Build](#build)
-  - [Running on Kubernetes](#running-on-kubernetes)
 - [Deployment](#deployment)
-  - [Harbor 2.0 on Kubernetes](#harbor-20-on-kubernetes)
+  - [Harbor >= 2.0 on Kubernetes](#harbor--20-on-kubernetes)
```

Comment:
Maybe it's just me but this link didn't resolve for me for some reason.

Reply:
I double checked with Chrome and Safari and it works 😕